cmd/geth/chaincmd.go

```diff
@@ -144,6 +144,17 @@ It's deprecated, please use "geth db import" instead.
 		Description: `
 The export-preimages command exports hash preimages to an RLP encoded stream.
 It's deprecated, please use "geth db export" instead.
+`,
+	}
+	exportOverlayPreimagesCommand = &cli.Command{
+		Action:    exportOverlayPreimages,
+		Name:      "export-overlay-preimages",
```

Comment:
```suggestion
		Name:      "export-preimages",
```
I mean, it's just exporting preimages, right?